### PR TITLE
added S3 bucket resource

### PR DESCRIPTION
**Pull request by Kurt812**

### CODE REVIEW

File: terraform/terraform.tfstate.backup [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
[the remark this answers is not in the document]
Comment (Kurt812) [reply terraform/terraform.tfstate.backup]: This has been fixed.

File: .gitignore [hunk 2]
```diff
@@ -8,3 +8,5 @@
 .terraform
 *.tfstate
 *.csv
+*.tfstate.*
```
Comment (S1mpySloth) [inline .gitignore:11]: Discussed on extra gitignore files and added `*.tfstate.*` and `.terraform.*` to exclude a broader range

File: terraform/main.tf [hunk 3]
```diff
@@ -18,3 +18,11 @@ resource "aws_ecr_repository" "c14_team_growth_lmnh" {
 output "ecr_repository_id" {
   value = aws_ecr_repository.c14_team_growth_lmnh.id
 }
+
+resource "aws_s3_bucket" "gamma_long_term_storage" {
```
Comment (S1mpySloth) [inline terraform/main.tf:22]: Could you make the bucket name something more descriptive for our project? i.e. c14-team-growth-gamma-long-term-storage

Comment (Kurt812) [reply terraform/main.tf:22]: I feel like that is a bit long, could it be c14-team-growth-storage?

Comment (S1mpySloth) [reply terraform/main.tf]: Yes, even better!

**Pull request merged**